testing/tests/DevExpress.common/charts.tests.js

```diff
@@ -1,4 +1,5 @@
-import { registerPattern, registerGradient, getGraphicObjects } from 'common/charts';
+import { registerPattern, registerGradient } from 'common/charts';
+import graphicObjects from '__internal/common/m_charts';
```

Comment:
Could we import registerPattern and registerGradient from '__internal/common/m_charts'?

Reply:
common/charts - it's a public path to this methods. I think we should testing this methods using public paths(or how we check, what public methods is working)

Reply:
ok, could we import { getGraphicObjects } there instead of graphicObjects?

Reply:
we have
export default { getGraphicObjects };
this kind of export was added for [mock](https://github.com/DevExpress/DevExtreme/blob/23_2/testing/tests/DevExpress.viz.charts/pieChart.tests.js#L2438) this function. if we will not export { getGraphicObjects }, we can't write this mock.